modules/038-registry/docs/FAQ.md

```diff
@@ -97,7 +97,7 @@ If such configurations exist:
 {% alert level="danger" %}
 - This switch is only possible from the `Unmanaged` mode.
 - When switching to the legacy `Containerd V1` auth configuration, any custom configurations in `/etc/containerd/registry.d` will stop working.
-- [Custom auth configurations](/products/kubernetes-platform/documentation/v1/modules/node-manager/faq.html#how-to-add-additional-registry-auth) for the legacy auth format (using `/etc/containerd/conf.d`) can only be applied after switching to the legacy mode.
```

Comment:
How user can swicth to legacy mode?
It looks like we're don't want user switch to legacy mode.